test/colors.mjs

```diff
@@ -0,0 +1,58 @@
+'use strict';
```

Comment:
Quick question, was this used to generate the screenshots?

Adding a comment to the top of this file to explain what it's for would be useful.
I would also move this to `etc/colors.mjs`. @baileympearson does that sound right?

Reply:
This is what was used to generate the screenshots, I can add a comment to the top of this, and also move it to etc/colors.mjs once we hear back from Bailey

Reply:
If we decide to keep this file, `etc` seems appropriate.  But if this was just used to generate screen shots for the release, I'm not sure it provides much value and we can remove it.

Reply:
I'll go ahead and remove the file.